pkg/resources/stream.go

```diff
@@ -190,15 +190,15 @@ func CreateStream(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return err
 	}
-	d.SetId(dataIDInput)
+	data.SetId(dataIDInput)
 
-	return ReadStream(d, meta)
+	return ReadStream(data, meta)
 }
 
 // ReadStream implements schema.ReadFunc
-func ReadStream(d *schema.ResourceData, meta interface{}) error {
+func ReadStream(data *schema.ResourceData, meta interface{}) error {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R014: *schema.ResourceData parameter of CreateFunc, ReadFunc, UpdateFunc, or DeleteFunc should be named d